### PR TITLE
RELATED: RAIL-590 KPI handles empty data correctly

### PR DESCRIPTION
**Pull request by zbycz**

### CODE REVIEW

File: src/helpers/errorHandlers.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ export function convertErrors(error: Execution.IError) {
     }
 }
 
+/** @deprecated */
 function isNullExecutionResult(responses: Execution.IExecutionResponses): boolean {
     return responses.executionResult === null;
 }
```
Comment (zbycz) [inline src/helpers/errorHandlers.ts]: Can really be @deprecated ?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/299/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/222/ : SUCCESS in 2m 38s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/312/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/233/ : SUCCESS in 2m 50s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-merge-compliance-check/103/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/234/ : SUCCESS in 2m 40s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/52/ : SUCCESS in 3m 24s

**Pull request merged**